docker/install-scripts/base.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 set -e
 
-apk update
-apk add git curl openssl
-apk add py-pip
-apk add --virtual=build gcc libffi-dev musl-dev openssl-dev python3-dev make
+apt update
+apt install -y git curl openssl
+apt install -y pip
```

Comment:
We are no longer using pip?

Reply:
It still looks like we use pip in https://github.com/XenitAB/github-actions/blob/f36fee3d4c17e1806a34f65a7c56d1113215589b/docker/install-scripts/yq.sh
Other then that no. 
If possible to find a ubuntu registry that we trust where we can download yq we should be able to save allot of spcae.